connectors/google/install.js

```diff
@@ -28,31 +28,30 @@ fusebit_allowed_return_to=${ctx.body.configuration.fusebit_allowed_return_to}
             files: {
                 'package.json': {
                     engines: {
-                        node: "10"
+                        node: '10',
                     },
                     dependencies: {
-                        "@fusebit/add-on-sdk": "^0.0.1",
-                        "superagent": "^5.2.2",
-                        "googleapis": "39"
-                    }
+                        '@fusebit/add-on-sdk': '^0.0.1',
```

Comment:
Fusebit add-on-sdk version needs updating.

Reply:
Won't fix, I'm only working on Slack in this change (with the exception of prettifying the other addons)